platform.txt

```diff
@@ -107,9 +109,14 @@ recipe.S.o.pattern="{compiler.path}{compiler.S.cmd}" {compiler.S.flags} {build.u
 archive_file_path={build.path}/{archive_file}
 recipe.ar.pattern="{compiler.path}{compiler.ar.cmd}" {compiler.ar.flags} {compiler.ar.extra_flags} "{archive_file_path}" "{object_file}"
 
-## Combine gc-sections, archives, and objects
+## Generate the linker map with specific flash sizes/locations
```

Comment:
changes taken by https://github.com/earlephilhower/arduino-pico/pull/143 , which has been merged but not updated by github. In my experience, you need to temporarily change the target to other branch, then set it back to master for github to re-diff.

Reply:
Yes, looks fine when I pull the PR locally and `git diff master`.